Domain/Aggreagtes/StudentAggregate/Student.cs

```diff
@@ -16,18 +18,20 @@ public class Student : AuditableEntity, IAggregateRoot
         public DateOnly? DateOfBirth { get; private set; }
         public Address? Address { get; private set; }
         public EducationLevel? Education { get; private set; }
-
         public Guid? CourseId { get; private set; }
         public virtual Course? Course { get; private set; }
+        public Guid UserId { get; private set; } = default!;
+        public virtual User User { get; private set; }
 
         public string Fullname => $"{Firstname}{Lastname}";
-        internal Student(string studentNumber, string firstname, string lastname, string phoneNumber, string emailAddress)
+        internal Student(string studentNumber, string firstname, string lastname, string phoneNumber, string emailAddress,DateOnly dateOfBirth)
```

Comment:
A construstor is used to defined a minimum that data required for an object of a class to be valid. Why including DOB in the constructor? Can't there be a student without DOB?